www/docs/en/dev/guide/platforms/android/plugin.md

```diff
@@ -23,24 +23,17 @@ title: Android Plugins
 # Android Plugins
 
 This section provides details for how to implement native plugin code
-on the Android platform. Before reading this, see Application Plugins
+on the Android platform. Before reading this, see the [Plugin Development Guide][plugin-dev]
```

Comment:
I am okay with anything as long as it's consistent. 'Android Development Guide' and 'Android Plugin Development Guide' and so on for each platform?